ring-core/test/ring/assets/blog.chp

```diff
@@ -0,0 +1 @@
+[:div "I'm a blog"]
```

Comment:
There are already plenty of generic files for testing. You don't need to add one more.